src/test/init/features/firestore.spec.ts

```diff
@@ -6,7 +6,7 @@ import { FirebaseError } from "../../../error";
 import * as firestore from "../../../init/features/firestore";
 import * as indexes from "../../../init/features/firestore/indexes";
 import * as rules from "../../../init/features/firestore/rules";
-import * as requireAccess from "../../../requireAccess";
+import * as rP from "../../../requirePermissions";
```

Comment:
nit: strange camel-case shorthand.  Don't think we do that anywhere else?

Reply:
replaced